File: app/src/main/java/com/example/esl/models/local/entities/LaporanMasalah.kt
package com.example.esl.models.local.entities

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "laporanmasalah")
data class LaporanMasalah(
    @PrimaryKey(autoGenerate = true) val id_laporan: Int = 0,
    val id_users: Int,
    val id_penyewaan: Int,
    val masalah: String,
    val media: String?,
    val tanggal_laporan: String
)

File: models/laporanmasalah.js
'use strict';
const {
  Model
} = require('sequelize');
module.exports = (sequelize, DataTypes) => {
  class LaporanMasalah extends Model {
    /**
     * Helper method for defining associations.
     * This method is not a part of Sequelize lifecycle.
     * The `models/index` file will call this method automatically.
     */
    static associate(models) {
      LaporanMasalah.belongsTo(models.Users, { foreignKey: 'id_users' });
      LaporanMasalah.belongsTo(models.Penyewaan, { foreignKey: 'id_penyewaan' });
    }
  }
  LaporanMasalah.init({
    id_laporan: {
      type: DataTypes.INTEGER,
      primaryKey: true,
      autoIncrement: true,
    },
    id_users: DataTypes.INTEGER,
    id_penyewaan: DataTypes.INTEGER,
    masalah: DataTypes.TEXT,
    media: DataTypes.STRING,
    tanggal_laporan: DataTypes.DATE
  }, {
    sequelize,
    modelName: 'LaporanMasalah',
  });
  return LaporanMasalah;
};